src/plugins/content_management/common/rpc/get.ts

```diff
@@ -9,7 +9,16 @@ import { schema } from '@kbn/config-schema';
 import type { Version } from '@kbn/object-versioning';
 import { versionSchema } from './constants';
 
-import type { ProcedureSchemas } from './types';
+import type { ItemResult, ProcedureSchemas } from './types';
+
+export const getResultSchema = schema.object(
+  {
+    contentTypeId: schema.string(),
+    // --> "result" will be (optionally) specified by each storage layer
+    result: schema.object({}, { unknowns: 'allow' }),
```

Comment:
I am not sure where exactly, but should this somewhere mention `item` from `ItemResult` ?

Reply:
Good point. Fixed in https://github.com/elastic/kibana/pull/154150/commits/6baf668605c489b37f97534513cf4586bce75d76